apps/coscrad-frontend/src/app/app.tsx

```diff
@@ -60,6 +60,10 @@ const subscribeToRealTimeUpdates = (dispatch: ReturnType<typeof useAppDispatch>)
             const { type } = event;
 
             // TODO Map the view collection names to aggregate types in the repository layer
```

Comment:
Oh I see what this is getting at. The notification service should be able to convert the database collection name to an `aggregateType`, the latter being the correct representation at this level of abstraction.